spec/controllers/users_controller_spec.rb

```diff
@@ -1,8 +1,46 @@
 require "rails_helper"
 
+def get_with_authorization(path, options)
+  options[:headers] ||= {
+    "Accept": "application/vnd.api+json",
```

Comment:
looks like the Users controller has been changed to not use jsonapi format anymore, is this header still needed?